src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,420 @@
+            });
+    };
+
+    function waitingServiceDestroy(uuid: string, timeout: number, date: Date) {
+        setTip(
+            MigrateSubmitResult(
+                'Destroying, Please wait... [' +
+                    Math.ceil((new Date().getTime() - date.getTime()) / 1000).toString() +
+                    's]',
+                uuid,
+                'success'
+            )
+        );
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        ServiceService.getDeployedServiceDetailsById(uuid, localStorage.getItem(usernameKey)!)
+            .then((response) => {
+                if (response.serviceState === ServiceDetailVo.serviceState.DESTROY_SUCCESS) {
+                    setTip(
+                        MigrateSubmitResult(
+                            ProcessingStatus(response, OperationType.Destroy as OperationType),
+                            uuid,
+                            'success'
+                        )
+                    );
+                    setIsDestroying(true);
+                    getMigrateModalCloseStatus(false);
+                    getMigrateModalOpenStatus(false);
+                    return () => {
+                        clearTimeout(retryDestroyTimer);
+                    };
+                } else if (response.serviceState === ServiceDetailVo.serviceState.DESTROY_FAILED) {
+                    setTip(
+                        MigrateSubmitResult(
+                            ProcessingStatus(response, OperationType.Destroy as OperationType),
+                            uuid,
+                            'error'
+                        )
+                    );
+                    setIsDestroying(false);
+                    getMigrateModalCloseStatus(true);
+                    return () => {
+                        clearTimeout(retryDestroyTimer);
+                    };
+                } else {
+                    retryDestroyTimer = window.setTimeout(() => {
+                        waitingServiceDestroy(uuid, timeout - waitServicePeriod, date);
+                    }, waitServicePeriod);
+                }
+            })
+            .catch((error: Error) => {
+                console.log('waitingServiceDestroy error', error);
+                if (timeout > 0) {
+                    retryDestroyTimer = window.setTimeout(() => {
+                        waitingServiceDestroy(uuid, timeout - waitServicePeriod, date);
+                    }, waitServicePeriod);
+                } else {
+                    setTip(MigrateSubmitFailed(error));
+                    setIsDestroying(false);
+                    getMigrateModalCloseStatus(true);
+                    TipClear();
+                }
+            });
+    }
+
+    const steps = [
+        {
+            title: 'Export data',
+            content: <ExportServiceData />,
+            description: 'Export service data.',
+        },
+        {
+            title: 'Select a destination',
```

Comment:
ensure that the next button is enabled after deployment is successful.

Reply:
fixed.